doc/release-notes/release-notes-3.1.0.md

```diff
@@ -18,6 +18,8 @@ Canopy will activate on mainnet at height 1046400.
 
 See [ZIP 251](https://zips.z.cash/zip-0251) for additional information about the deployment process for Canopy.
 
+Debian8 "Jessie" will no longer be supported after v3.1.0 due to EOS.
```

Comment:
This is a bit ambiguous.

```suggestion
Debian 8 "Jessie" will no longer be supported after v3.1.0, due to its [end-of-life](https://www.debian.org/News/2020/20200709#:~:text=The%20Debian%20Long%20Term%20Support,security%20updates%20for%20Debian%208) on June 30th, 2020. This will allow us to direct more resources to supporting Debian 10 Buster, other Linux distributions, and other platforms such as Windows and macOS.
```